core/src/main/resources/hudson/logging/LogRecorderManager/all.jelly

```diff
@@ -31,6 +31,9 @@ THE SOFTWARE.
   <st:include page="sidepanel.jelly" />
   <l:main-panel>
     <h1><l:icon class="icon-clipboard icon-xlg"/>${%Jenkins Log}</h1>
+    <div class="alert alert-info">
+      Log messages at a level lower than INFO are never recorded in the Jenkins log. Use <a href="/">log recorders</a> to record these log messages.
```

Comment:
```suggestion
      Log messages at a level lower than INFO are never recorded in this default Jenkins log. Use <a href="/">log recorders</a> to record these log messages.
```

Reply:
I think the link shouldn't go to `/`

Reply:
Good point, this seems wrong. I think I wanted it to go to `.` due to how weird the entire logging UI is structured, which would be a straightforward typo.